src/session/cookie-store/index.ts

```diff
@@ -60,12 +84,17 @@ export default class CookieSessionStore implements ISessionStore {
       persistedSession.refreshToken = refreshToken;
     }
 
+    if (this.settings.storeRefreshToken && refreshToken && expiresAt) {
```

Comment:
I think storing the `expiresAt` should be unrelated to the refresh token being present or not. The expiration of the access token can be tracked always, even if we don't store the RT.